docs/features/telemetry-enrichment.md

```diff
@@ -73,4 +73,43 @@ ILogger logger = new LoggerConfiguration()
 logger.Information("This event will be enriched with the application component's name");
 ```
 
+## Correlation Enricher
+
+The `Arcus.ObservabilityTelemetry.Serilog.Enrichers` library provides a [Serilog enricher](https://github.com/serilog/serilog/wiki/Enrichment)
+that adds the `CorrelationInfo` information from the current context as log properties with the names `OperationId` and `TransactionId`.
+
+You can use your own `ICorrelationInfoAccessor` implementation to retrieve this `CorrelationInfo` model,
+or use the default `DefaultCorrelationInfoAccessor` implementation that stores this model
+
+**Example**
+Name: `OperationId`
+Value: `52EE2C00-53EE-476E-9DAB-C1234EB4AD0B`
+
+Name: `TransactionId`
+Value: `0477E377-414D-47CD-8756-BCBE3DBE3ACB`
+
+**Usage**
+
+```csharp
+ICorrelationInfoAccessor defaultImplementation = new DefaultCorrelationInfoAccessor();
+
+ILogger logger = new LoggerConfiguration()
+    .Enrich.WithCorrelationInfo(defaultImplementation)
```

Comment:
Seeing this, I think we should provide an overload `.Enrich.WithCorrelationInfo()` that uses `DefaultCorrelationInfoAccessor` behind the scenes

Reply:
Ah, yes, thank you.

Reply:
Hmm, the problem exists that we don't have access anymore to that default accessor. Unless you want me to make it a public available singleton (`DefaultCorrelationInfoAccessor.Instance`).

Reply:
Done just that.

Reply:
What have you done? Provide an overload where you don't have to pass it or?

If so, please update the docs here.